structured/db.go

```diff
@@ -67,13 +67,7 @@ func (db *structuredDB) PutSchema(s *Schema) error {
 
 // DeleteSchema removes s from the kv store.
 func (db *structuredDB) DeleteSchema(s *Schema) error {
-	return db.kvDB.Run(client.Call{
-		Args: &proto.DeleteRequest{
-			RequestHeader: proto.RequestHeader{
-				Key: engine.MakeKey(engine.KeySchemaPrefix, proto.Key(s.Key)),
-			},
-		},
-		Reply: &proto.DeleteResponse{}})
+	return db.kvDB.Run(client.Delete(engine.MakeKey(engine.KeySchemaPrefix, proto.Key(s.Key))))
```

Comment:
This is actually the old client API. The new one, which I'm steadily transitioning all the code towards, is in `client/db.go`. I'm about to send out a change which transitions some of the codebase to use it. I'll cc you so you can see what needs to be done here.